File: 101-109-smallproblems/easy8/middlechar.rb

def center_of(str)
  chomped = str
  while chomped.length > 2
    chomped = chomped[1..-2]
  end
  chomped
end

## Tests

p center_of('I love ruby') == 'e'
p center_of('Launch School') == ' '
p center_of('Launch') == 'un'
p center_of('Launchschool') == 'hs'
p center_of('x') == 'x'

File: 210/small-problems/easy5/middle-char.js
function centerOf(string) {
  var center = string.length / 2;

  return string.substring(Math.ceil(center) - 1, Math.floor(center) + 1);
}

console.log(centerOf('I Love Ruby'));      // "e"
console.log(centerOf('Launch School'));    // " "
console.log(centerOf('Launch'));           // "un"
console.log(centerOf('Launchschool'));     // "hs"
console.log(centerOf('x'));                // "x"
